lib/src/commit.rs

```diff
@@ -101,15 +102,21 @@ impl Commit {
             .collect()
     }
 
-    pub fn tree(&self) -> Result<MergedTree, BackendError> {
+    pub fn tree(&self) -> BackendResult<MergedTree> {
         self.store.get_root_tree(&self.data.root_tree)
     }
 
     pub fn tree_id(&self) -> &MergedTreeId {
         &self.data.root_tree
     }
 
-    pub fn has_conflict(&self) -> Result<bool, BackendError> {
+    /// Return the parent tree, merging the parent trees if there are multiple
+    /// parents.
+    pub fn parent_tree(&self, repo: &dyn Repo) -> BackendResult<MergedTree> {
```

Comment:
There's one more caller in cli/src/diff_util.rs

Reply:
Thanks, fixed.